src/utils.zig

```diff
@@ -362,6 +362,10 @@ pub fn CircularBufferQueue(comptime T: type) type { // MARK: CircularBufferQueue
 			}
 		}
 
+		pub inline fn enqueue_front(self: *Self, elem: T) void {
```

Comment:
I agree that the current variant is not quite optimal. But instead of aliasing it I would prefer to only have one way to do things. (of course this should be done in a separate PR, and it's fine to keep it like this here, though I would appreciate making an issue or adding a TODO note here for future reference)

Reply:
#1320